pyproject.toml

```diff
@@ -18,6 +18,8 @@ openpyxl = "^3.0.9"
 rich = "^12.1.0"
 beautifulsoup4 = "^4.11.1"
 structlog = "^21.5.0"
+owid-datautils = {git = "https://github.com/owid/data-utils-py.git", rev = "v0.4.4-alpha"}
+owid-catalog = "^0.2.9"
```

Comment:
Do we need to explicitly add this? It should be installed as dependency of `owid-datautils`

Reply:
Weird, perhaps installing from `git` doesn't pick up poetry requirements. Anyway, good workaround ;).